[APM][.NET] 2.10.0 update (#14249)
DataDog/documentation: content/en/tracing/setup_overview/setup/dotnet-framework.md

`DD_TRACE_SAMPLE_RATE`
: **TracerSettings property**: `GlobalSamplingRate` <br>
**Default**: `1.0`<br>

> Enables ingestion rate control. The sampling rate for the traces (defaults to: between `0.0` and `1.0`).

Reviewer: Same question here.

> Enables ingestion rate control. The sampling rate for the traces (defaults to: between `0.0` and `1.0`).

Reviewer: ```suggestion
Enables ingestion rate control and specifies a the sampling rate for traces. Valid values are from `0.0` to `1.0`.
```

`DD_TRACE_SAMPLING_RULES`
: **TracerSettings property**: `datadog.trace.sampling_rules`<br>
**Default**: `null`<br>

> A JSON encoded string to configure the sampling rate. Examples: Set the sample rate to 20%: `'[{"sample_rate": 0.2}]'`. Set the sample rate to 10% for services starting with 'a' and span name 'b' and set the sample rate to 20% for all other services: `'[{"service": "a.*", "name": "b", "sample_rate": 0.1}, {"sample_rate": 0.2}]'` (see [Integration names](#integration-names)). Note that the JSON object **must** be included in single quotes (`'`) to avoid problems with escaping of the double quote (`"`) character.|

Reviewer 2: ```suggestion
A JSON encoded string to configure the sampling rate. Examples: Set the sample rate to 20%: `'[{"sample_rate": 0.2}]'`. Set the sample rate to 10% for services starting with 'a' and span name 'b' and set the sample rate to 20% for all other services: `'[{"service": "a.*", "name": "b", "sample_rate": 0.1}, {"sample_rate": 0.2}]'` (see [Integration names](#integration-names)). Note that the JSON object **must** be included in single quotes (`'`) to avoid problems with escaping of the double quote (`"`) character.
```

`DD_TRACE_GLOBAL_TAGS`
: **TracerSettings property**: `GlobalTags`<br>
If specified, adds all of the specified tags to all generated spans.
**Example**: `layer:api, team:intake` <br>

> Note that the delimiter is a comma and a whitespace: `, `.

Reviewer: ```suggestion
Note that the delimiter is a comma and a space: `, `.
```

Added in version 2.6.0.
Wildcard support `[*]` added in version 2.7.0.

`DD_TRACE_KAFKA_CREATE_CONSUMER_SCOPE_ENABLED`

> : Alters the behaviour of the consumer span<br>
> When set to true, the consumer span is created and its duration is representative to the computation between one message consumption up to the next one. Prefer this setting when message consumption and processing are synchronous.
> When set to false, the consumer span is still created but only for the purpose of storing Kafka tags, its duration isn't representative. Prefer this setting when message consumption and processing are asynchronous.
> **Default**: `true`

Reviewer 2: ```suggestion
: Alters the behaviour of the kafka consumer span<br>
**Default**: `true`
When set to `true`, the consumer span is created when a message is consumed and closed before consuming the next message. The span duration is representative of the computation between one message consumption and the next. Prefer this setting when message consumption is performed in a loop.
When set to `false`, the consumer span is created when a message is consumed and immediately closed. Prefer this setting when a message is not processed completely before consuming the next one, or when multiple messages are consumed at once. 
```

> **Default**: `true`
> When set to `true`, the consumer span is created when a message is consumed and closed before consuming the next message. The span duration is representative of the computation between one message consumption and the next. Prefer this setting when message consumption is performed in a loop.
> When set to `false`, the consumer span is created when a message is consumed and immediately closed. Prefer this setting when a message is not processed completely before consuming the next one, or when multiple messages are consumed at once. 

Reviewer: ```suggestion
**Default**: `true`<br>
When set to `true`, the consumer span is created when a message is consumed and closed before consuming the next message. The span duration is representative of the computation between one message consumption and the next. Use this setting when message consumption is performed in a loop. <br>
When set to `false`, the consumer span is created when a message is consumed and immediately closed. Use this setting when a message is not processed completely before consuming the next one, or when multiple messages are consumed at once. 
```
